Scripts/carthage.sh

```diff
@@ -16,4 +16,4 @@ echo 'EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_simulator__NATIVE_ARCH_64_BIT_x8
 echo 'EXCLUDED_ARCHS = $(inherited) $(EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_$(EFFECTIVE_PLATFORM_SUFFIX)__NATIVE_ARCH_64_BIT_$(NATIVE_ARCH_64_BIT)__XCODE_$(XCODE_VERSION_MAJOR))' >> $xcconfig
 
 export XCODE_XCCONFIG_FILE="$xcconfig"
-carthage "$@"
+"${SRCROOT}/bin/carthage" "$@"
```

Comment:
Please revert; we should not be checking in the `carthage` binary into SRCROOT.

Reply:
I agree that I generally wouldn't want to commit binaries here, but DIY builds need the carthage bin pinned; they will not build on a m1 mac with newer Carthage for now. It's a convenience I hope we can do away with soon, if the carthage issues are fixed, or we move to support SPM.